src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -1,29 +1,76 @@
 package traffic_simulation
 
+// Imports for parsing library
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import java.io.*
+
 fun main(args: Array<String>) {
 
-    scenario(testList, testRoad)
+    val testRoad : RoadNetwork = RoadNetwork(capacity = 8.0)
+
+    val driveInterest : MutableList<Vehicle> = parseInputOfCSV(fileName = "driveInterest.csv")
```

Comment:
Die Datei gehört in `src/main/resources/driveInterest.csv` statt `src/ressources/driveInterest.csv` (ein Ordner tiefer, "s" im Ordnernamen).